core/src/main/scala/kafka/controller/Election.scala

```diff
@@ -17,11 +17,12 @@
 package kafka.controller
 
 import kafka.api.LeaderAndIsr
+import kafka.utils.Logging
 import org.apache.kafka.common.TopicPartition
 
 case class ElectionResult(topicPartition: TopicPartition, leaderAndIsr: Option[LeaderAndIsr], liveReplicas: Seq[Int])
 
-object Election {
+object Election extends Logging {
```

Comment:
how often will this object be instantiated? reason im asking is we've been burned by the Logging aspect on short lived objects before

Reply:
It's an object so it should  only be instantiated once.